### PR TITLE
[docs][YSQL] CREATE AGGREGATE, DROP AGGREGATE

### PR DESCRIPTION
**Pull request by jj-kim**
Closes issue #2227.

### CODE REVIEW

File: docs/content/latest/api/ysql/syntax_resources/ysql_grammar.ebnf [hunk 1]
```diff
@@ -538,3 +612,21 @@ param_name ::= '<Text Literal>' ;
 param_value ::= '<Text Literal>' ;
 
 code ::= '<Text Literal>' ;
+
+aggregate_argmode = 'IN' | 'VARIADIC' ;
```
Comment (ndeodhar) [inline docs/content/latest/api/ysql/syntax_resources/ysql_grammar.ebnf]: ::= 
Same for others below.

File: docs/content/latest/api/ysql/commands/ddl_create_aggregate.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+---
+title: CREATE AGGREGATE
+linkTitle: CREATE AGGREGATE
+summary: Create a new type in a database
```
[the remark this answers is not in the document]
Comment (jj-kim) [reply docs/content/latest/api/ysql/commands/ddl_create_aggregate.md]: Nice catch.

File: docs/content/latest/api/ysql/syntax_resources/ysql_grammar.ebnf (hunk 1 above)
Comment (jj-kim) [reply docs/content/latest/api/ysql/syntax_resources/ysql_grammar.ebnf]: See issue #2308 for the move towards `=`.  Feel free to add comments there if you still prefer `::=`.

File: docs/content/latest/api/ysql/syntax_resources/ysql_grammar.ebnf [hunk 3]
```diff
@@ -69,6 +116,76 @@ copy_option ::=
...
+                       'SFUNC' '=' sfunc ','
+                       'STYPE' '=' state_data_type
+                       { ',' aggregate_old_option } ')' ;
+
+aggregate_arg = [ aggregate_argmode ] [ argname ] argtype ;
+
+aggregate_normal_option = 'SSPACE' '=' state_data_size
+                        | 'FINALFUNC' '=' ffunc
+                        | 'FINALFUNC_EXTRA'
+                        | 'FINALFUNC_MODIFY' '=' { 'READ_ONLY' | 'SHAREABLE' | 'READ_WRITE' }
```
[the remark this answers is not in the document]
Comment (jj-kim) [reply docs/content/latest/api/ysql/syntax_resources/ysql_grammar.ebnf]: Oops, copy/paste.

**Pull request merged**